Source/AwsCommonRuntimeKit/http/HttpProxyEnvSettings.swift

```diff
@@ -5,11 +5,11 @@ import AwsCHttp
 public struct ProxyEnvSettings: CStruct {
```

Comment:
the filename changed ~ProxyEnvSettings.swift~ -> `HttpProxyEnvSettings.swift`
but the class name didn't change

Reply:
Thanks, good catch.